StatisticsTests/Histograms/HistogramTests.cs

```diff
@@ -211,7 +211,8 @@ public void NormallyDistributed_Histogram_CentralTendency(int n, double binWidth
         [InlineData(.1,1)]
         public void IsHistogramConstructableWithNullData(double binWidth, double expected)
         {
-            Histogram histogram = new Histogram(null, binWidth);
+            double observationValue = 0.5;
+            Histogram histogram = new Histogram(observationValue, binWidth);
```

Comment:
this is no longer using histogram with null data, i think we set this back to null right?

Reply:
the histogram should not be constructed using null data with the Histogram(data, binwidth) constructor, right? that doesn't really make sense to me. I'd rather delete the test.

Reply:
yeah i see what you are saying, i dont disagree. but if we detect data is null, it is constructed with the binwidth set. as long as we dont call "min or max" on a null array we havent corrupted data nor created an improper data behavior, nor triggered a null pointer. id rather drop the array constructor all together.